lib/src/services/pub_service.dart

```diff
@@ -36,6 +37,8 @@ class PubService {
   /// Checks whether or not has the latest version for `stacked_cli` package
   /// installed on the system.
   Future<bool> hasLatestVersion() async {
+    locator<ColorizedLogService>()
+        .stackedOutput(message: 'Before get current version', isBold: true);
```

Comment:
Why did you add this? 🤔  The function should only check if has or not the latest version. No output needed.

Reply:
Forgot to remove it, thanks.